unpackaged/config/rd2_post_config/objects/npe03__Recurring_Donation__c.object

```diff
@@ -268,6 +268,16 @@
             </valueSetDefinition>
         </valueSet>
     </fields>
+    <fields>
```

Comment:
@andrewyu-salesforce  Since this is a new field being added to the object, it will need to be in the packaged version of the object file vs. the unpackaged object. This (unpackaged) object file is only used to override field help text and picklist active/inactive states.